Source/JavaScriptCore/inspector/remote/cocoa/RemoteConnectionToTargetCocoa.mm

```diff
@@ -203,6 +203,9 @@ static void RemoteTargetHandleRunSourceWithInfo(void* info)
 void RemoteConnectionToTarget::close()
 {
     Locker locker { m_targetMutex };
+
+    m_closed = true;
```

Comment:
we may also want to do this in `RemoteConnectionToTarget::targetClosed` just in case